core/docker/build.sh

```diff
@@ -9,6 +9,8 @@ Builds the Trino Docker image
 
 -h       Display help
 -a       Build the specified comma-separated architectures, defaults to amd64,arm64,ppc64le
+-p       Use the specified server package (artifact id), defaults to trino-server
```

Comment:
Add that other possible value is trino-server-core

Reply:
@nineinchnick let me know when you updated this .. I will merge after

Reply:
Done